monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/micro/integrator/delegates/DataSourcesDelegate.java

```diff
@@ -37,14 +37,18 @@
  */
 public class DataSourcesDelegate implements ArtifactDelegate {
 
-    private static final Log log = LogFactory.getLog(DataSourcesDelegate.class);
-    private final DatabaseManager databaseManager = DatabaseManagerFactory.getDbManager();
+    private static final Log logger = LogFactory.getLog(DataSourcesDelegate.class);
 
     @Override
-    public Artifacts getArtifactsList(String groupId, List<String> nodeList) {
-
-        log.debug("Fetching data-sources from database.");
-        return databaseManager.fetchArtifacts(Constants.DATA_SOURCES, groupId, nodeList);
+    public ArtifactsResourceResponse getPaginatedArtifactsResponse(String groupId, List<String> nodeList, 
+        String searchKey, String lowerLimit, String upperLimit, String order, String orderBy, String isUpdate) 
+        throws ManagementApiException {
+            
+        logger.debug("Fetching Searched data-sources from MI.");
+        logger.info("group id :" + groupId + ", lowerlimit :" + lowerLimit + ", upperlimit: " + upperLimit);
```

Comment:
Do we need to print info logs everytime we fetch them?

Reply:
if so, shall i make it a debug log or simply remove?

Reply:
@arunans23 This is fixed with https://github.com/wso2/product-mi-tooling/pull/310/commits/14d2304ff9c7a633f82a2f687a1137ee9968cf55

Reply:
Let's keep it as debug logs.

Reply:
fixed